testpilot/frontend/static-src/app/views/feedback-view.js

```diff
@@ -7,78 +6,39 @@ export default BaseView.extend({
     <div class="modal-container">
       <div id="{{id}}-modal" class="modal feedback-modal modal-bounce-in">
         <h3 class="title" data-l10n-id="{{title}}"></h3>
-        <form>
-          <div class="modal-content modal-form">
-            <span data-l10n-id="feedbackErrorMessage" class="form-notification" data-hook="form-notification">Please choose an option.</span>
-            <ul class="questions">
-              {{#questions}}
-              <li>
-                <input type="radio"
-                  name="{{id}}-question"
-                  id="{{id}}-question-{{value}}"
-                  value="{{value}}">
-                <label data-hook="title"
-                  for="{{id}}-question-{{value}}"
-                  data-l10n-id="{{title}}"></label>
-              </li>
-              {{/questions}}
-            </ul>
-            <div class="additional-feedback">
-              <p data-l10n-id="feedbackExtraCaption" class="text-area-label">Do you have any additional feedback? (optional)</p>
-              <textarea data-l10n-id="feedbackExtraTextarea"
-                placeholder="what's on your mind..."></textarea>
-            </div>
+        <div class="modal-content">
+          <div class="copter-wrapper fly-down">
+            <div class="copter"></div>
           </div>
+          <p class="centered" data-lion-id="feedbackUninstallCopy">
```

Comment:
autocomplete? You using MS Word to write JavaScript?

<img width="316" alt="untitled" src="https://cloud.githubusercontent.com/assets/557895/14647268/c88dc7d8-0611-11e6-871b-88077784d2c4.png">